.github/workflows/test.yaml

```diff
@@ -0,0 +1,21 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        runner: [ 'ubuntu-18.04', 'ubuntu-20.04', 'ubuntu-latest' ]
```

Comment:
FYI: For the moment ubuntu-20.04 == ubuntu-latest, but this setup means the tests will start running on ubuntu-22.04 (or any other more recent version) as soon as it's available.